tools/test-tools/src/getTestPort.ts

```diff
@@ -7,27 +7,37 @@ import fs from "fs";
 import os from "os";
 import path from "path";
 
+const defaultPort = 8081;
+
 /**
  * Get the port for the pkg from the mapping.  Use a default if the file or the entry doesn't exist
  * (e.g. an individual test is being run and the file was never generated), which should presumably
  * not lead to collisions.
  */
-export function getTestPort(pkgName: string): string {
-	let mappedPort: string;
+export function getTestPort(pkgName: string): number {
+	let mappedPort: number;
+
+	const commonErrorString = `Using default port ${defaultPort}. If you encounter port collisions, be sure to run assign-test-ports.`;
```

Comment:
Nit: this could be hoisted out of the function.

Reply:
Done!